plugins/cpu/fu-cpu-helper-cet.c

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2020 Richard Hughes <richard@hughsie.com>
+ * Copyright (C) 2020 H.J. Lu <hjl.tools@gmail.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <signal.h>
+#include <stdlib.h>
+
+#include "fu-cpu-helper-cet-helper.h"
```

Comment:
cpu-helper-cet-helper?  I think this should have a better name

Reply:
fu-cpu-helper-cet-common?

Reply:
Yeah, just avoid helper-*-helper